.github/workflows/run-all-tests.yml

```diff
@@ -60,17 +60,6 @@ jobs:
         run: .github/scripts/echoBuildBuddyConfig.sh ${{ secrets.BUILDBUDDY_API_KEY }} >> $GITHUB_ENV
         shell: bash
 
-      - name: Set CC to LLVM clang on macOS
```

Comment:
Is the macOS restriction in the matrix section further up still needed?

Reply:
Thanks, I don't think so. Removed it.